smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/plugins/DefaultPlugins.kt

```diff
@@ -0,0 +1,13 @@
+package software.amazon.smithy.swift.codegen.integration.plugins
+
+import software.amazon.smithy.swift.codegen.SwiftDependency
+import software.amazon.smithy.swift.codegen.SwiftWriter
+import software.amazon.smithy.swift.codegen.integration.Plugin
+
+enum class DefaultPlugins(override val className: String) : Plugin {
+    DefaultClientPlugin("DefaultClientPlugin") {
+        override fun addImport(writer: SwiftWriter) {
+            writer.addImport(SwiftDependency.CLIENT_RUNTIME.target)
+        }
+    }
+}
```

Comment:
Is using `enum` idiomatic in Swift? One alternative is creating a static list of plugins, since it is converted to a list in the `SwiftIntegration` anyways.

```swift
override fun plugins(): List<Plugin> {
    return DefaultPlugins.values().toList()
}
```